packages/itwin/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,117 @@
+{
+  "name": "@itwin/grouping-mapping-widget",
+  "version": "0.1.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/itwin/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/cjs/grouping-mapping-widget.js",
+  "module": "lib/esm/grouping-mapping-widget.js",
+  "typings": "lib/cjs/grouping-mapping-widget",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": ""
+  },
+  "dependencies": {
+    "@itwin/imodel-browser-react": "^0.12.1",
+    "@itwin/itwinui-css": "^0.44.1",
```

Comment:
https://github.com/iTwin/iTwinUI-react/wiki/version-conflicts

Reply:
Quick question, I put this in here due to the fear that PNPM was not fetching itwinui-reacts dependencies. Is there any reason to this? I know that getting the Scss variables from the transitive dependency is the way to go, but whenever I do rush install. It doesn't seem to install the dependency at all from itwinui-react.

Reply:
surprised rush isnt pulling it in. i would then try just setting it as dev dep then.

before you do that, try removing the direct dep, and doing a rush update --full; wonder if it does pull in the css pkg but not the right ver you were expecting

Reply:
Yup I did that after doing rush -purge. Still refuses.

Reply:
I'll just leave it as a dev dep.